browser/src/layer/tile/CommentSection.ts

```diff
@@ -754,6 +756,18 @@ export class Comment extends CanvasSectionObject {
 		}
 	}
 
+	// check if this is "our" autosaved comment
+	// core is not aware it's autosaved one so use this simplified detection based on content
+	public isAutoSaved (): boolean {
+		var autoSavedComment = cool.CommentSection.autoSavedComment;
+		if (!autoSavedComment)
+			return false;
+
+		var authorMatch = autoSavedComment.sectionProperties.data.author === this.sectionProperties.data.author;
+		var textMatch = autoSavedComment.sectionProperties.data.text === this.sectionProperties.data.text;
```

Comment:
This makes reply autosave fail in writer. 

When we reply to any comment we edit into the original comment element in reply textbox, so original comment is set as `autoSavedComment` value and `this.sectionProperties.data.text` is new reply, so these values are not going to match for reply.

Reply:
ok, then we should match only author